consensus/tangle/tangle.h

```diff
@@ -54,6 +54,10 @@ retcode_t iota_tangle_transaction_load_hashes_of_approvers(
     tangle_t const *const tangle, flex_trit_t const *const approvee_hash,
     iota_stor_pack_t *const pack);
 
+retcode_t iota_tangle_transaction_load_hashes_of_requests(
```

Comment:
documentation? :upside_down_face:

Reply:
Added, I'll write all other tangle docs in another PR.